examples/face_landmarker/ios/FaceLandmarker/Helper/FaceLandmarkerHelper.swift

```diff
@@ -0,0 +1,139 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import MediaPipeTasksVision
+import AVFoundation
+
+protocol FaceLandmarkerHelperDelegate: AnyObject {
+  func faceLandmarkerHelper(_ faceLandmarkerHelper: FaceLandmarkerHelper,
+                             didFinishDetection result: ResultBundle?,
+                             error: Error?)
+}
+
+class FaceLandmarkerHelper: NSObject {
+
+  weak var delegate: FaceLandmarkerHelperDelegate?
+  var faceLandmarker: FaceLandmarker?
+
+  init(modelPath: String?, numFaces: Int, minFaceDetectionConfidence: Float, minFacePresenceConfidence: Float, minTrackingConfidence: Float, runningModel: RunningMode, delegate: FaceLandmarkerHelperDelegate?) {
+    super.init()
+    guard let modelPath = modelPath else { return }
+    let faceLandmarkerOptions = FaceLandmarkerOptions()
+    faceLandmarkerOptions.runningMode = runningModel
+    faceLandmarkerOptions.numFaces = numFaces
+    faceLandmarkerOptions.minFaceDetectionConfidence = minFaceDetectionConfidence
+    faceLandmarkerOptions.minFacePresenceConfidence = minFacePresenceConfidence
+    faceLandmarkerOptions.minTrackingConfidence = minTrackingConfidence
+    faceLandmarkerOptions.faceLandmarkerLiveStreamDelegate = runningModel == .liveStream ? self : nil
+    self.delegate = delegate
+    faceLandmarkerOptions.baseOptions.modelAssetPath = modelPath
+    do {
+      faceLandmarker = try FaceLandmarker(options: faceLandmarkerOptions)
+    } catch {
+      print(error)
+    }
+  }
+
+  /**
+   This method return FaceLandmarkerResult and infrenceTime when receive an image
```

Comment:
"This method returns a FaceLandmarkerResult object and infrenceTime after receiving an image"